### PR TITLE
 Add single-server-with-existing-network example 

### PR DESCRIPTION
**Pull request by dschwabeS11**
- How to launch an instance using an existing network

### CODE REVIEW

File: reserve_floating_ips/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Reserve Floating IPs
+# single server with existing network
```
Comment (bpetermannS11) [inline reserve_floating_ips/README.md]: is this change intended?

Comment (dschwabeS11) [reply reserve_floating_ips/README.md]: uh, no!

File: single-server-with-existing-network/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Parameters
+
+**public_network**  
+References the external network connected to the internet.
+
+**net**  
+Defines ID of the network to connect the instance port to.
+
+**image**  
+Image to be used to build instance.
```
Comment (bpetermannS11) [inline single-server-with-existing-network/README.md]: to build the instance

File: single-server-with-existing-network/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+$ openstack stack create -t example.yaml -e example-env.yaml <new stackName>
+```
+
+## Parameters
+
+**public_network**  
+References the external network connected to the internet.
+
+**net**  
+Defines ID of the network to connect the instance port to.
```
Comment (bpetermannS11) [inline single-server-with-existing-network/README.md]: Defines the ID...

File: single-server-with-existing-network/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+**image**  
+Image to be used to build instance.
+
+**flavor**  
+Instance flavor to be used.
+
+**ssh-key**  
+SSH key to be injected into instance.
```
Comment (bpetermannS11) [inline single-server-with-existing-network/README.md]: ...into the instance

File: single-server-with-existing-network/example.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
+heat_template_version: 2016-04-08
+#
+# you can deploy this template using the following command:
+# 'openstack stack create -t example.yaml -e example-env.yaml <stackName>'
+# 
+# after successful stack creation you can login with
```
Comment (bpetermannS11) [inline single-server-with-existing-network/example.yaml]: ...can log in with
